root/static/scripts/edit/URLCleanup.js

```diff
@@ -645,6 +645,27 @@ const CLEANUPS = {
       return {result: false};
     },
   },
+  'appleco': {
+    match: [new RegExp('^(https?://)?([^/]+\\.)?apple\\.co/', 'i')],
+    type: LINK_TYPES.downloadpurchase,
```

Comment:
This works, but autoselecting to downloadpurchase, even if only to block the link, will mean that the user is likely to enter the redirect destination as downloadpurchase even if incorrect. So it might be better to wait with this.